### PR TITLE
Inclusion of last volunteer name who made a successful encounter for the patient

### PR DESCRIPTION
**Pull request by ialuj**
Inclusion of last volunteer name who made a successful encounter for the patient PR

### CODE REVIEW

File: app/src/main/java/com/muzima/view/patients/PatientSummaryActivity.java [hunk 1]
```diff
@@ -40,20 +41,8 @@
...
-import com.muzima.api.model.Concept;
-import com.muzima.api.model.Encounter;
-import com.muzima.api.model.EncounterType;
-import com.muzima.controller.ConceptController;
-import com.muzima.controller.ObservationController;
-import com.muzima.controller.FormController;
-import com.muzima.controller.EncounterController;
-import com.muzima.controller.MuzimaSettingController;
-import com.muzima.controller.PatientController;
+import com.muzima.api.model.*;
```
Comment (benamonya) [inline app/src/main/java/com/muzima/view/patients/PatientSummaryActivity.java:44]: import individual not  by *

File: app/src/main/java/com/muzima/view/patients/PatientSummaryActivity.java [hunk 2]
```diff
@@ -292,8 +282,24 @@ private void loadPatientData() {
 
             conceptController = ((MuzimaApplication) getApplicationContext()).getConceptController();
             observationController = ((MuzimaApplication) getApplicationContext()).getObservationController();
-            encounterController =  ((MuzimaApplication) getApplicationContext()).getEncounterController();
+            EncounterController encounterController =  ((MuzimaApplication) getApplicationContext()).getEncounterController();
             formController =  ((MuzimaApplication) getApplicationContext()).getFormController();
+            DerivedConceptController derivedConceptController = ((MuzimaApplication) getApplicationContext()).getDerivedConceptController();
+            DerivedObservationController derivedObservationController = ((MuzimaApplication) getApplicationContext()).getDerivedObservationController();
+
+            // 31ba65ef-4d23-414c-bb9a-1aefce4c2d5c
```
Comment (benamonya) [inline app/src/main/java/com/muzima/view/patients/PatientSummaryActivity.java:290]: remove the commented out uuid

File: app/src/main/java/com/muzima/view/patients/PatientSummaryActivity.java [hunk 3]
```diff
@@ -689,15 +699,18 @@ private void initializeResources() {
...
         TabLayout tabLayout = findViewById(R.id.tabLayout);
         LinearLayout dadosDeConsentimento = findViewById(R.id.dados_de_consentimento);
+        LinearLayout detalhesDaUltimaVisita = findViewById(R.id.general_last_encounter_details);
         RelativeLayout addressLayout = findViewById(R.id.address_layout);
         RelativeLayout phoneNumberLayout = findViewById(R.id.phone_number_layout);
         RelativeLayout artInitDateLayout = findViewById(R.id.date_of_art_start_layout);
 
         if (!isFGHCustomClientSummaryEnabled) {
             dadosDeConsentimento.setVisibility(View.GONE);
+            //detalhesDaUltimaVisita.setVisibility(View.GONE);
```
Comment (benamonya) [inline app/src/main/java/com/muzima/view/patients/PatientSummaryActivity.java:713]: remove this commented out code

File: app/src/main/java/com/muzima/view/patients/PatientSummaryActivity.java (hunk 2 above)
Comment (ialuj) [reply app/src/main/java/com/muzima/view/patients/PatientSummaryActivity.java]: Comment removed

File: app/src/main/java/com/muzima/view/patients/PatientSummaryActivity.java (hunk 3 above)
Comment (ialuj) [reply app/src/main/java/com/muzima/view/patients/PatientSummaryActivity.java]: Comment removed

**Review by benamonya — APPROVED** [review]

**Pull request merged**